src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/communityapi/Conviction.kt

```diff
@@ -30,3 +31,32 @@ data class Custody(
 data class CustodyStatus(
   val code: String
 )
+
+data class Offence @JsonCreator constructor(
```

Comment:
We have 2 x `Offence` objects - one in this class and one in CaseSummaryOverviewResponse.kt, is that correct or can they be combined? When should you use one over the other?

Reply:
The Offence object in this class is in the package 'communityapi'  
indicating that it comes from the community api's conviction endpoint. 

There is also a package for responaes/requests to the communityapi's offender-search endpoint.

All domain objects outside of these packages are responses from our make-recall-decisions-api.

I made two copies as one represents the offence object that the community-api serves up.
The other represents the offence object that our api serves.

I made two in as I see each having a different responsibility.
I'm happy to make a change and have the community-api/our api share if you like :)

Reply:
Sounds good - guess if it gets confusing we could always append one of those 2 objects with a different name e.g. ` OffenceResponse`